### PR TITLE
https://www.pivotaltracker.com/story/show/150258169

### PR DESCRIPTION
**Pull request by kendelljoseph**

### CODE REVIEW

**Review by grng00ber — APPROVED** [review]
Looks good

File: .gitignore [hunk 1]
```diff
@@ -23,15 +23,15 @@ coverage
...
-
 # node-waf configuration
 .lock-wscript
 
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 build/Release
 
+# Bower components
+bower_components/
+
```
Comment (kendelljoseph) [inline .gitignore:34]: Don't worry about this

**Comment by deeptronos** [comment]
Looks correct!

**Review by deeptronos — APPROVED** [review]
Yeaaah

**Pull request merged**